po/ja.po

```diff
@@ -16817,18 +16852,24 @@ msgstr ""
 #: src/concurrency/send-sync/examples.md:36
 msgid "These types are not thread-safe and cannot be moved to other threads:"
 msgstr ""
+"こうした型はスレッドセーフではないため、別のスレッドにムーブすることはできま"
+"せん："
 
 #: src/concurrency/send-sync/examples.md:38
 msgid ""
 "`Rc<T>`: each `Rc<T>` has a reference to an `RcBox<T>`, which contains a non-"
 "atomic reference count."
 msgstr ""
+"`Rc<T>`: それぞれの `Rc<T>` は、ノンアトミックな参照カウントを含む`RcBox<T>`"
+"への参照を持っています。"
```

Comment:
I think "Rc<T>はRcBox<T>への参照を持っています。これは、アトミックでない参照カウントを持っています" is better.
Your translation may sound that `RcBox<T>` value that doesn't have non-atomic reference count can exist.
In other words, this sentence should be translated as 「関係代名詞の継続用法」instead of 「関係代名詞の制限用法」.

Reply:
Oh yes, I didn't notice comma here which can change the nuance. Thank you!